nncf/quantization/fake_quantize.py

```diff
@@ -230,10 +231,12 @@ def calculate_quantizer_parameters(
     else:
         num_bits = quantizer_config.num_bits
         if quantizer_config.mode == QuantizationMode.SYMMETRIC:
-            _, _, levels = calculate_symmetric_level_ranges(num_bits, signed=True, narrow_range=narrow_range)
+            level_low, level_high = calculate_symmetric_level_ranges(num_bits, signed=True, narrow_range=narrow_range)
+            levels = get_num_levels(level_low, level_high)
```

Comment:
What do you think about such simplification?
```suggestion
            levels = calculate_levels(num_bits, narrow_range=narrow_range)
```

Reply:
The signatures are different for the asymmetric and symmetric cases, I could introduce such a function but I doubt that there would be much savings in the number of code lines.

Reply:
On develop:
`def calculate_symmetric_level_ranges(num_bits: int, signed: bool, narrow_range: bool = False) -> Tuple[int, int, int]:`
`def calculate_asymmetric_level_ranges(num_bits: int, narrow_range: bool = False) -> Tuple[int, int, int]:`

Reply:
The calculation levels function does not depend on signed parameter, isn't it?